content/en/docs/Responses/DirectPlayProfile.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "DirectPlayProfile"
+linkTitle: "DirectPlayProfile [OS]"
+opensubsonic:
+- Extension
+description: >
+  Defines a direct play profile.
+---
+
+{{< tabpane persist=false >}}
+{{< tab header="**Example**:" disabled=true />}}
+{{< tab header="OpenSubsonic" lang="json">}}
+{
+  "container": "mp3",
+  "audioCodec": "mp3",
+  "protocol": "http",
+  "maxAudioChannels": 2
+}
+{{< /tab >}}
+{{< /tabpane >}}
+
+| Field | Type | Req. | OpenS. | Details |
+| --- | --- | --- | --- | --- |
+| `container` | string | **Yes** | **Yes** | The container format (e.g., mp3, flac, mp4). |
+| `audioCodec` | string | **Yes** | **Yes** | Comma-separated list of supported audio codecs. |
```

Comment:
Why a comma-separated list, instead of an array?

Reply:
Because the others do it like that and because of "*", but can be an array, just need to be clear about don't use * and something else.

Reply:
I think we should then write down a note to say it is either "*" or a comma-separated list of audio codec names (and * is forbidden in the latter case)